lib/modules/manager/quadlet/extract.ts

```diff
@@ -54,7 +55,13 @@ export function extractPackageFile(
 ): PackageFileContent | null {
   const deps: PackageDependency<Record<string, any>>[] = [];
 
-  const quadletFile: QuadletFile = Ini.pipe(QuadletFile).parse(content);
+  const res = QuadletFile.safeParse(content);
+  if (!res.success) {
+    logger.debug({ err: res.error }, 'Error parsing Quadlet file.');
```

Comment:
```suggestion
    logger.debug({ err: res.error, packageFile }, 'Error parsing Quadlet file.');
```

So it's easier to see which file failed